tests/test_policy.py

```diff
@@ -297,6 +297,8 @@ def test_config_resource_support(self):
                 "Missing config types \n %s" % ('\n'.join(missing)))
 
         bad_types = resource_config_types.difference(config_types)
+        # TODO: remove once botocore is updated https://github.com/boto/botocore/issues/2111
```

Comment:
its already present, the pinned version in custodian needs updating

Reply:
o really. i tried with botocore version 1.17.26 and still got the same error as sns topic was missing. ill look again and see if i missed something. thanks

Reply:
1.17.27 is released with the fix, i've got a separate pr to update the deps open.

Reply:
if you rebase on trunk, it should work, and then you can remove the whitelist entry in the config test_policy.py  coverage test

Reply:
awesome thanks!